_posts/2023-05-05-NASA-grant-announcement.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "One small step for NASA, one big step for open research: OLS and MetaDocencia to deliver OpenCore Science Training"
+authors:
+- yochannah
+---
+
+For many, 2023 is the [Year of Open Science](https://www.whitehouse.gov/ostp/news-updates/2023/01/11/fact-sheet-biden-harris-administration-announces-new-actions-to-advance-open-and-equitable-research/). 
+
+OLS and MetaDocencia (MD) are delighted to announce that we will be participating in the Year of Open Science by delivering Open Science training for NASA, using materials originally developed by [a huge group of open researchers worldwide](https://opensciency.github.io/sprint-content/#contributors), alongside AGU.
+
+## What will we do? 
+
+Spanish is the second most spoken language in the United States, after English. When NASA launched the grant call, MD & OLS partnered to propose a dual Spanish/English language set of training cohorts using our expertise in online virtual cohort training and Spanish-language education. [We are delighted to share that we've been awarded a three-year grant to deliver this training!](https://www.nasa.gov/centers/marshall/news/releases/2023/nasa-boosts-open-science-through-innovative-training/)
+
+The training will cover [the five modules in OpenCore](https://opensciency.github.io/sprint-content), and participants will work with a mentor to discuss real-world application of the skills they use. The schedule for individual cohorts is shown below: 
+
+![Gantt chart showing five training modules taking 2.5 hours each, as well as 30 minutes of mentor discussion, and an hour of self-reflection each week, totalling 4 hours per week for six weeks. The final week is a live-streamed graduation presentation.](https://hackmd.io/_uploads/Hk-H2tEb2.png)
```

Comment:
It would be more accessible to have this image as a table in the text directly. Is it possible?

Reply:
@NPDebs can you make it so? 🙏  let me know if having access to the table in google sheets would help

Reply:
Yes, please. That would help a lot, @yochannah. Thanks!

Reply:
sent to you!